### PR TITLE
Add support for iTerm2

### PR DESCRIPTION
**Pull request by YusukeSano**
The [terminal.sexy](https://terminal.sexy/) provides iTerm2 color schemes via [.Xresources](https://gist.github.com/cocopon/1d481941907d12db7a0df2f8806cfd41), but some settings, such as Selection and Link, are not supported.

So I created a complete [Icebreg for iTerm2](https://github.com/YusukeSano/iterm2-iceberg) and added port links.

|Before|After|
|---|---|
|||
<br>

This also supports separate color settings for light & dark modes, which will be added in the latest beta (3.5) of iTerm2.

|Light|Dark|
|---|---|
|||

### CODE REVIEW

**Comment by jf** [comment]
hi, just a fan of iceberg here, looking to use it for iTerm2 too. Do you think you could give a brief summary of how your version compares versus https://github.com/epszaw/iceberg-iterm (from https://github.com/cocopon/iceberg.vim/pull/80) ? Thank you!

**Comment by YusukeSano** [comment]
> hi, just a fan of iceberg here, looking to use it for iTerm2 too. Do you think you could give a brief summary of how your version compares versus https://github.com/epszaw/iceberg-iterm (from #80) ? Thank you!

Sure! Here's a summary of [my color theme file](https://github.com/YusukeSano/iterm2-iceberg) compared to the #80 version of the theme:

-   Compatibility: My theme files are designed to support light & dark mode since version 3.5.
-   Consistency: It aligns with the original [.Xresources](https://gist.github.com/cocopon/1d481941907d12db7a0df2f8806cfd41) file for a consistent color experience.
-   Enhanced Colors: I have referenced the [VSCode theme file](https://github.com/cocopon/vscode-iceberg-theme) to enrich the color palette.
-   Accurate Representation: The theme file was created using the precise sRGB color values from the [iTerm2 Theme Generator](https://github.com/YusukeSano/iterm2-theme-generator).

In summary, my version provides improved compatibility, consistent colors, enhanced palette, and accurate representation.

**Comment by jf** [comment]
thank you @YusukeSano ! @cocopon , do you think you could take a look and see if you're ok with this PR? Thank you!

**Comment by YusukeSano** [comment]
iTerm2 version 3.5 has been released! (https://iterm2.com/downloads.html)

Version 3.5 supports color profiles for both light and dark modes.
Additionally, a new `find match background` color has been added.

Unfortunately, [terminal.sexy](https://terminal.sexy) and #80 do not support the correct terminal colors.
However, [Icebreg for iTerm2](https://github.com/YusukeSano/iterm2-iceberg) fully supports all these updates.

|light|dark|
|---|---|
|light|dark|

Would you kindly review this pull request, @cocopon?

**Comment by gkeep** [comment]
Can confirm, looks and works great. Thanks, @YusukeSano!

**Comment by cocopon** [comment]
Thank you for your contribution! (also thanks to @gkeep for reviewing it!)

**Comment by jf** [comment]
thank you everybody! @cocopon I previously (see https://github.com/cocopon/iceberg.vim/pull/105#issuecomment-1556186937) asked @YusukeSano to compare his work vs #80, and he has kindly done so. If you've accepted this PR, would you also reject #80?
